classes/Http/Controller/Admin/SpeakersController.php

```diff
@@ -122,9 +123,15 @@ public function viewAction(Request $req)
 
     public function deleteAction(Request $req)
     {
+        $csrfTokenManager = $this->service('csrf.token_manager');
```

Comment:
What do you think about

* injecting services (rather than the container) so we can more easily write test
* sorting this out with a middleware instead

?

Reply:
No objections, can you point me at an example?

Reply:
I‘ll see if I can find out, but I remember @BackEndTea just recently moved authentication to a middleware, so I think it should be possible.

Reply:
I read some comment in i think the controller service provider that we cant really inject services the normal way because we override some of that. So fixing that would allow the service injection.

Reply:
So are we going to wait until we fix this or are you folks okay with pushing these commits because it fixes a security hole? We can definitely create an issue surrounding this

Reply:
I'm okay with pushing this to fix a security hole, creating an issue and then resolving that later.